lib/ASTStmt.cpp

```diff
@@ -417,6 +436,52 @@ namespace pillar
 
       return CreateDo(cond_expr, body);
     }
+
+    clang::DeclStmt *AST::LiftVarDeclOp(clang::DeclContext *dc, mlir::Operation &op_)
+    {
+      auto var_decl = LiftVarDeclOp(dc, dc, mlir::dyn_cast<vast::hl::VarDeclOp>(op_));
+      return new (ctx) clang::DeclStmt(clang::DeclGroupRef(var_decl), kEmptyLoc,
+                                       kEmptyLoc);
+    }
+
+    clang::Expr *AST::LiftInitListExpr(clang::DeclContext *dc, mlir::Operation &op_)
+    {
+      vast::hl::InitListExpr op = mlir::dyn_cast<vast::hl::InitListExpr>(op_);
+
+      mlir::Operation::operand_range elements = op.getElements();
+
+      std::vector<clang::Expr *> expressions;
+
+      for (mlir::Value operand : elements)
+      {
+        mlir::Operation *definingOp = operand.getDefiningOp();
```

Comment:
Use `expressions.push_back(LiftValue(operand));`. The rest shouldn't be needed. The key thing to remember is that by the time you've gotten to where you are, you've probably lifted everything you'll need, and so if you have an `mlir::Value`, then to get the lifted "thing" for that, use `LiftValue`.